http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/RuntimeServiceInterp.scala

```diff
@@ -91,6 +96,31 @@ class RuntimeServiceInterp[F[_]: Parallel](config: RuntimeServiceConfig,
             _ <- context.span.traverse(s => F.delay(s.addAnnotation("Done Sam getAccessToken")))
             clusterImages <- getRuntimeImages(petToken, context.now, req.toolDockerImage, req.welderDockerImage)
             _ <- context.span.traverse(s => F.delay(s.addAnnotation("Done get runtime images")))
+            diskOpt <- req.diskConfig.traverse(x =>
+              processDiskConfigRequest(x,
+                                       googleProject,
+                                       req.runtimeConfig.map(_.cloudService).getOrElse(CloudService.GCE),
+                                       userInfo)
+            )
+            _ <- context.span.traverse(s => F.delay(s.addAnnotation("Done process disk config")))
```

Comment:
oh nice...I was gonna say that we can add some tracing for new routes now that tracing PR is merged